config/secrets.yml

```diff
@@ -0,0 +1,16 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rake secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
+  secret_key_base: '54dec6e791dce377c9b3a28aca520b427be2b469d3f4f4632158435a7edf0e3267fadaf94520d5dd5910116a227d6c7904bc81164de4f97a6549aea144dec46d'
+test:
+  secret_key_base: '54dec6e791dce377c9b3a28aca520b427be2b469d3f4f4632158435a7edf0e3267fadaf94520d5dd5910116a227d6c7904bc81164de4f97a6549aea144dec46d'
```

Comment:
This should have a line for production, reading from ENV, (heres an [example from asset-manager](https://github.com/alphagov/asset-manager/commit/6f7c5971c7e6b080c40e1aa573be047fc31cf2cf)

It'll need a change in `puppet`/`deployment`, which i'm looking at, to populate the ENV.